redis/src/main/scala/zio/redis/api/Connection.scala

```diff
@@ -74,6 +74,32 @@ trait Connection {
     command.run(())
   }
 
+  /**
+   * Returns information about the current connection.
+   *
+   * @return
+   *   the current connection information in property=value fields separated by a space character.
+   */
+  final def clientInfo: ZIO[Has[Redis], RedisError, ClientInfo] = {
+    val command = RedisCommand(ClientInfo, NoInput, ClientInfoOutput)
+
+    command.run(())
+  }
+
+  /**
+   * Returns information about the client connections.
+   *
+   * @param filter
+   *   subcomand to filter the clientList by client type or client ids.
+   * @return
+   *   the list of clients information in property=value fields separated by a space character.
+   */
+  final def clientList(filter: Option[ClientListFilter] = None): ZIO[Has[Redis], RedisError, Chunk[ClientInfo]] = {
+    val command = RedisCommand(ClientList, ClientListInput, ClientListOutput)
+
+    command.run(filter.getOrElse(ClientListFilter.All))
```

Comment:
Maybe set `ClientListFilter.All` as default, and drop options completely.

Reply:
you're right, I had not noticed :)